### DISCUSSION TITLE
Is there something similar to setDefaultEdgeLabel in Dagre.NET?

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [fel88/Dagre.NET#1] [Ideas]
Feedback 1: Provide a link for new User of Dagre.NET  to dagre Wiki

https://github.com/dagrejs/dagre/wiki

Question 1: Is there similar setDefaultEdgeLabel function in Dagre.NET.  If so, a small example how to use that.

```js
// Default to assigning a new object as a label for each new edge.
g.setDefaultEdgeLabel(function() { return {}; });
```

**Comment by fel88** [comment]
> Is there similar setDefaultEdgeLabel function in Dagre.NET.

Not yet. But I can add it.

PS: It seems that setDefaultEdgeLabel was never referenced in the dagre.js
